app/views/includes/organisation-overview.html

```diff
@@ -1,3 +1,16 @@
+<table class="export-buttons">
+    <tr>
+        <td>
+        {% ifAsync ((organisationLevel === 'region') or (organisationLevel === 'ldu') or (organisationLevel === 'team')) %}
+        {% if ((authorisation===false) or (userRole==='Data Admin') or (userRole==='Manager') or (userRole==='System Admin')) %}
```

Comment:
I don't think user role should affect reduction exports. All users should be able to export from region, ldu or team level